<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="AddA">
     <attribute name="title">
      <string>      Ajouter     </string>
     </attribute>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_typean">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_codean">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">ui-&gt;LineEdit_Name-&gt;setStyleSheet(&quot;QLineEdit{ &quot;
                                        &quot;background-color:rgb(202, 255, 227);&quot;
                                        &quot;border: 2px solid gray;&quot;
                                        &quot;border-radius: 10px;&quot;
                                        &quot;padding: 0 8px;&quot;
                                        &quot;selection-background-color: darkgray;&quot;
                                        &quot;font-size: 16px;}&quot;
                                        &quot;QLineEdit:focus { &quot;
                                        &quot;background-color:rgb(192, 192, 255);}&quot;
                                        );</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Code de l'annonce</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pays">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Destination</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nbplace">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Nombre de place</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idcompagne">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Identifiant de la compagne</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>280</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Load types</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="addT">
     <attribute name="title">
      <string>Ajouter / Supprimer un type </string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>351</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout type</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_type">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Nouveau type</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>281</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: yellow;
    selection-background-color: darkgray;
}</string>
       </property>
       <property name="documentTitle">
        <string/>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::WidgetWidth</enum>
       </property>
       <property name="placeholderText">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_type">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>260</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>351</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Suppression type</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_typean_supp">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Type à supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer_type">
       <property name="geometry">
        <rect>
         <x>114</x>
         <y>160</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modif">
     <attribute name="title">
      <string>     Modifer     </string>
     </attribute>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_codean_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Code de l'annonce à modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pays_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Destination</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nbplace_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Nombre de place</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_typean_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idcompagne_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Identifiant de la compagne</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix_modif">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>280</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Load types</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="supp">
     <attribute name="title">
      <string>     Supprimer    </string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>511</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Suppression</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_codean_2">
       <property name="geometry">
        <rect>
         <x>122</x>
         <y>120</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Code de l'annonce à supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>164</x>
         <y>170</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="affiche">
     <attribute name="title">
      <string>     Afficher      </string>
     </attribute>
     <widget class="QTableView" name="tabannonce">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>621</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>340</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="trier">
     <attribute name="title">
      <string>Trier selon PRIX</string>
     </attribute>
     <widget class="QTableView" name="tabannonce_tri">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>621</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_asc_tri">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>330</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ascendant </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_desc_tri">
      <property name="geometry">
       <rect>
        <x>404</x>
        <y>330</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Descendant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pays">
     <attribute name="title">
      <string>Destination spécifiée</string>
     </attribute>
     <widget class="QTableView" name="tabannonce_aff_pays">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>611</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_aff_pays">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Destination</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_aff_pays">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>70</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
